### PR TITLE
Feature/microej sdk 6 publish executable

### PR DESCRIPTION
**Pull request by denisbedin**

### CODE REVIEW

File: SDK6UserGuide/troubleshooting.rst [hunk 1]
```diff
@@ -100,6 +100,27 @@ If the following message is displayed when publishing a module::
 it means the ``version`` property is missing and should be defined in the module build file.
 See :ref:`sdk_6_publish_project` for more information.
 
+Fail to load VEE Port as dependency
+-----------------------------------
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/troubleshooting.rst:104]:
```suggestion
Fail to load a VEE Port as dependency
-------------------------------------
```

File: SDK6UserGuide/troubleshooting.rst [hunk 2]
```diff
@@ -100,6 +100,27 @@ If the following message is displayed when publishing a module::
...
+
+When a VEE Port is defined as a dependency, the build of the project can fail with the following message::
+
+	> No 'release.properties' and 'architecture.properties' files found.
+  The given file <path/to/file> is not a VEE Port archive.
+
+If the dependency is a valid VEE Port, this error means that several artifacts of the VEE Port have been published
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/troubleshooting.rst:111]:
```suggestion
If the dependency is a valid VEE Port, this error probably means that several artifacts of the VEE Port have been published
```

File: SDK6UserGuide/troubleshooting.rst [hunk 3]
```diff
@@ -100,6 +100,27 @@ If the following message is displayed when publishing a module::
...
+with the ``default`` ivy configuration.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/troubleshooting.rst:112]:
```suggestion
with the ``default`` Ivy configuration.
```

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

**Comment by denisbedin** [comment]
@tdelhomenie-microej I updated the doc with your suggestions.

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**